docs/developer_guide.md

```diff
@@ -128,6 +133,14 @@ to be brought up locally (in a separate process):
 
 By default the NL server runs on port 6060.
 
+If you run into problems starting the server, try running these commands before restarting the server:
+```bash
+./run_test.sh --setup_python
+rm -rf ~/.datacommons
+rm -rf /tmp/datcom-nl-models
+rm -rf /tmp/datcom-models-dev
```

Comment:
Sorry, I might have given you the wrong command before. I think this should be `rm -rf /tmp/datcom-nl-models-dev`

Reply:
ah yes - thanks for catching that! fixed.